CHANGES.rst

```diff
@@ -174,6 +174,8 @@ tweakreg
 
 - Updated step arguments in the documentation. [#4723]
 
+- Added align_to_gaia processing as an option [#4599]
```

Comment:
This needs to go up at the top of the file, under the 0.16.1 release (0.16.0 has already gone out).

Reply:
Rebased to latest version of JWST.   Updated correct version of CHANGES with this message under v0.16.1 in commit 78f34b6